src/documentation/programming-guide.md

```diff
@@ -1456,4 +1458,202 @@ An example might be if your pipeline reads log records from an input file, and e
 
 ## <a name="triggers"></a>Working with triggers
 
-> **Note:** This guide is still in progress. There is an open issue to finish the guide ([BEAM-193](https://issues.apache.org/jira/browse/BEAM-193))
+> **NOTE:** This content applies only to the Beam SDK for Java. The Beam SDK for Python does not support triggers.
```

Comment:
I moved some stuff around to try and improve this. PTAL and see what you think once it's staged.